templates/game.html.twig

```diff
@@ -1,5 +1,12 @@
 {% extends "base.html.twig" %}
 
+{% block head %}
+	{{ parent() }}
+    {% if (ships|first).player.id != game.currentPlayer.id %}
+	    <meta http-equiv="refresh" content="5">
```

Comment:
High level engineering :grin:

Reply:
Can you set a value to known you intention?

`{% set isOponentTurn = (ships|first).player.id != game.currentPlayer.id %}`

Reply:
Fixed. I added the game player to the view variable.